DCScribe.Postgres/Documentation/configuration.Sample.yaml

```diff
@@ -0,0 +1,76 @@
+---
+# The following setup is for the logging system. For information on how to change this
+# and what other options are available see https://serilog.net/
+Serilog:
+  Using:
+    - Serilog.Sinks.File
+  MinimumLevel: 
+    Default: Information
+  Override: 
+    Microsoft: Warning
+    System: Warning
+  Enrich:
+    - FromLogContext
+    - WithProcessId
+    - WithThreadId
+  WriteTo:
+    - 
+      Name: File
+      Args:
+        path: logs/dcscribe.log
+# This is the configuration for each DCS GameServer. This Sample configuration contains
+# configuration for two separate servers. Edit the configuration below, including adding
+# or removing servers, to suit your setup.
+GameServers:
+  - # These dashes indicate a configuration record for a new GameServer
+    # Replace the following example fields with those used by your server and database
+    Name: RurouniJones Bestest Server
+    # Used in logging
+    ShortName: RBS
+    Database:
+      # Make sure you set your pg_hba.conf file if you are not connecting over localhost
+      # https://www.postgresql.org/docs/current/auth-pg-hba-conf.html
+      Host: rbs.example.com
+      Port: 5432
+      Name: rbs_database
+      Username: dcscribe
+      Password: this_is_a_password
+    # This is the configuration for connecting to the DCS-gRPC server
+    Rpc:
+      # Because DCS-gRPC runs inside the DCS process this should be your DCS server hostname
+      # or IP address
+      Host: rbs.example.com
+      # This is the default port, change it if you have changed the server-side one
+      Port: 50051
+    # Enable or disable the unit stream
+    UnitStream: true
+    # Set the time in seconds between task runs
+    TaskTimers:
+      - # Query the DCS server for a list of the airbases in the running mission
+        TaskName: ProcessAirbaseUpdates
+        Timer: 60
+      - # Query the DCS server for a list of the map marks in the running mission
+        TaskName: ProcessMarkPanelUpdates
+        Timer: 120
+  # You can specify multiple game servers and have one DCScribe instance handle all of them
+  # This section configures a second server. Delete everything below this line to remove.
+  - 
+    Name: RurouniJones Worstest Server
+    ShortName: RWS
+    Database:
+      Host: rws.example.com
+      Port: 5432
+      Name: rws_database
+      Username: dcscribe
+      Password: this_is_a_password
+    Rpc:
+      Host: rws.example.com
+      Port: 50051
+    UnitStream: false
+    TaskTimers:
+      - 
+        TaskName: ProcessAirbaseUpdates
+        Timer: 90
+      - 
+        TaskName: ProcessMarkPanelUpdates
```

Comment:
I think it is clearer for users to have an `Enabled: true / false` entry for each task rather than a list where presence means enabled. Therefore this section of the config would look like

```yaml
Tasks:
  RecordUnitPositions:
    Enabled: true,
    Timer: 5
  ProcessAirbaseUpdates:
    Enabled: true,
    Timer: 90
  ProcessMarkPanelUpdates:
    Enabled: true,
    Timer: 90
```

Having this setup also means we can have task specific configuration such as `PollRate` instead of `Timer` for the `RecordUnitPositions` if we think that is a better name or add some other config.

Reply:
The abstracted task setup with their own config properties is better, I agree.